src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -214,11 +217,29 @@ export class ViewMappingProfile extends Component {
 
     const isMARCRecord = existingRecordType === MARC_TYPES.MARC_BIBLIOGRAPHIC;
 
+    const updatesFieldMappingForMARCColumns = ['field', 'indicator1', 'indicator2', 'subfield'];
+
+    const rebderMARCTableView = () => {
+      switch (marcMappingOption) {
```

Comment:
Are you sure that `switch` is needed here?

Reply:
As for me, it's a little more readable and easy to extend with one option in case if needed. But If you think if/else is better here I don't mind to change it.

Reply:
I'd suggest creating a variable for the default set of columns
```
const defaultFieldMappingForMARCColumns = ['action', 'field', 'indicator1', 'indicator2',
    'subfield', 'subaction', 'data', 'position']
```
And then use it smth like this
```
const renderMARCTableView = () => {
  const fieldMappingForMARCColumns = {
    UPDATES: updatesFieldMappingForMARCColumns,
    ...
  };

  return (
    <MARCTableView
          columns={fieldMappingForMARCColumns[marcMappingOption] || defaultFieldMappingForMARCColumns}
          fields={mappingDetails?.marcMappingDetails}
     />
  )
}; 
```

Don't know if this solution is better than `if/else` - it's up to you, but I guess the `switch` operator is not quite suitable for 1 or 2 options, what do you think?